pennylane/fermi/fermionic.py

```diff
@@ -28,6 +30,9 @@ class FermiWord(dict):
     <FermiWord = '0+ 1-'>
     """
 
+    __numpy_ufunc__ = None
+    __array_ufunc__ = None
```

Comment:
Worth adding a small note why these are needed. (to make sure that reverse addition/multiplication is possible for custom objects and numpy arrays, correct?)

Reply:
yes, I'll add a comment!

Reply:
I wonder if this is something we will need to do for every interface ? I hope not, but that's a future problem